src/main/cli/cli.c

```diff
@@ -5426,6 +5426,9 @@ static void printPeripheralDmaoptDetails(dmaoptEntry_t *entry, int index, const
 
     if (entry->presenceMask) {
         uiIndex = timerGetNumberByIndex(index);
+        if (!(BIT(uiIndex) & entry->presenceMask)) {
```

Comment:
Noe you can use more descriptive (but equivalent):
```suggestion
        if (!(TIM_N(uiIndex) & entry->presenceMask)) {
```

Reply:
Done